configs/bigcommerce.json

```diff
@@ -1,37 +1,625 @@
 {
-  "index_name": "bigcommerce",
+  "index_name": "TESTING_INDEX",
   "start_urls": [
-    "https://support.bigcommerce.com/",
-    "https://support.bigcommerce.com/guides",
-    "https://support.bigcommerce.com/university",
-    "https://support.bigcommerce.com/documentation"
-  ],
+	{"url":"https://support.bigcommerce.com/articles/Learning/What-is-SEO","selectors_key": "guides"},
```

Comment:
Seems big and complicated, can't you do something simpler?
It seems each `selectors_key` is associated with a specific URL pattern.